string-manipulation-1/src/is-lower-cased.ts

```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word: string): boolean {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i].toLowerCase() !== word[i]) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
While this does work, we can actually check the whole word rather than going letter by letter. Since `toLowerCase` is called on a string, we can use the whole string such as:
`word === word.toLowerCase()`